ui/src/assets/common.scss

```diff
@@ -670,13 +670,13 @@ button.query-ctrl {
 .track-group-panel {
   --collapsed-background: var(--main-background-color);
   --collapsed-transparent: fade-out(var(--main-background-color),1);
-  color: var(--main-foreground-color);
+  --expanded-background: var(--main-foreground-color);
   display: grid;
   grid-template-columns: auto 1fr;
   grid-template-rows: 1fr;
   transition: background-color 0.4s, color 0.4s;
   height: 40px;
-
+  
```

Comment:
ESLint squawks when a TS file has blank lines that aren't empty. Does the same happen for CSS files?